packages/reports/src/reportBuilder/functions/index.ts

```diff
@@ -0,0 +1,16 @@
+import { value, add, eq, neq, exists, gt } from './basic';
+import { convertToPeriod, periodToTimestamp, periodToDisplayString } from './utils';
+
+export const functions = {
+  value,
+  add,
+  eq,
+  neq,
+  gt,
+  exists,
+  convertToPeriod,
+  periodToTimestamp,
+  periodToDisplayString,
+};
```

Comment:
I think the idea with exporting collections of functions like this is that we can easily group them by functionality/scope. For now we've just got one general scope but I think this may change in the future